client/state/jetpack-connect/reducer.js

```diff
@@ -52,17 +52,30 @@ export function jetpackConnectSessions( state = {}, action ) {
 	switch ( action.type ) {
 		case JETPACK_CONNECT_STORE_SESSION:
 			return Object.assign( {}, state, buildNoProtocolUrlObj( action.url ) );
-		case SERIALIZE:
+		case JETPACK_CONNECT_SSO_AUTHORIZE_SUCCESS:
```

Comment:
damn.. between yesterday & today, I merged 9 different branches modifying this files and it has been a nightmare of conflicts and weird rebases ... sorry, I'll remove it